packages/nuxeo-web-ui-ftest/package.json

```
@@ -5,14 +5,19 @@
     "nuxeo-web-ui-ftest": "./bin/nuxeo-web-ui-ftest.js"
   },
   "scripts": {
+    "postinstall": "check-engine",
     "publish:snapshot": "npm --registry=https://mavenin.nuxeo.com/nexus/repository/npmjs-nuxeo/ publish --tag SNAPSHOT",
     "publish:release": "npm --registry=https://registry.npmjs.org/ publish"
   },
   "license": "Apache-2.0",
+  "engines": {
+    "node": ">=10.23.0"
```

Don't think this runs on node 12, because of the Fibers version that is currently installed. We should probably set something like:
```suggestion
    "node": ">=10.23.0 < 11"
```